src/api/Tag.php

```diff
@@ -15,7 +15,7 @@ class Tag
     const RESOURCE_NAME = 'resource.name';
     const DB_STATEMENT = 'sql.query';
     const ERROR = 'error';
-    const ERROR_MSG = 'error.msg'; // string representing the error message
+    const ERROR_MESSAGE = 'error.message'; // string representing the error message
```

Comment:
I think I would leave `ERROR_MSG` that points to the updated `error.message` name, with a comment about it being deprecated and to use `ERROR_MESSAGE` instead. This would prevent breakage for custom integrations users have written.

```suggestion
    /** @deprecated Use ERROR_MESSAGE instead. */
    const ERROR_MSG = 'error.message';
    const ERROR_MESSAGE = 'error.message'; // string representing the error message
```